client/starwhale/base/bundle.py

```diff
@@ -76,9 +76,29 @@ def list(
         project_uri: URI,
         page: int = DEFAULT_PAGE_IDX,
         size: int = DEFAULT_PAGE_SIZE,
+        filters: t.Union[t.Dict[str, t.Any], t.List[str]] = {},
     ) -> t.Tuple[t.Dict[str, t.Any], t.Dict[str, t.Any]]:
         _cls = cls._get_cls(project_uri)
-        return _cls.list(project_uri, page, size)  # type: ignore
+        _filter = cls.get_filter_dict(filters, cls.get_filter_fields())
+        return _cls.list(project_uri, page, size, _filter)  # type: ignore
+
+    @classmethod
+    def get_filter_dict(
+        cls, filters: t.Union[t.Dict[str, t.Any], t.List[str]], fields: t.List[str] = []
```

Comment:
fields that use mutable default value will meet the same issue as filters in 79 line.

Reply:
`fields: t.List[str] = []`
-->
`fields: t.Optional[t.List[str]] = None`
+ 
`fields = fields or None`

Reply:
done